### PR TITLE
Running MPI within a LAN

### PR DESCRIPTION
**Pull request by dtsdwarak**
Updated the repo for running MPI within a LAN. With minor modifications to install instructions.

### CODE REVIEW

File: tutorials/installing-mpich2/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ MPICH2 F77:
...
\ No newline at end of file
+Now that you have built MPICH2 locally, you have a couple options of where you can proceed on this site. If you don't have access to a cluster or want to learn more about building a virtual MPI cluster, check out the lesson about [building and running your own cluster on Amazon EC2]({{ site.baseurl }}/tutorials/launching-an-amazon-ec2-mpi-cluster/). If you already have your own cluster or simply want to run the rest of the lessons from your machine, proceed to the [MPI hello world lesson]({{ site.baseurl }}/tutorials/mpi-hello-world/), which provides an overview of the basics of programming and running your first MPI program.
```
Comment (wesleykendall) [inline tutorials/installing-mpich2/index.md]: Can you add in a note about the guide on running from a LAN here? I try to point people in the directions of the next tutorials at the end of the tutorial

File: tutorials/running-a-mpi-cluster-within-lan/index.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+```
+
+Or specify the same in a hostfile and 
+
+```bash
+$ mpirun -np 5 --hostfile mpi_file ./cpi
+```
+
+This should spin up your program in all of the machines that your **master** is connected to. 
+
```
Comment (wesleykendall) [inline tutorials/running-a-mpi-cluster-within-lan/index.md:196]: Can you provide a section here to guide people to another tutorial? I generally try to do an "Up next" or something like that and then point the user to another tutorial (or give them a link back to the beginner tutorial page)

**Comment by wesleykendall** [comment]
@dtsdwarak looks great! I had a couple comments. Feel free to add yourself in the Authors section as well (http://mpitutorial.com/about/) with a blurb about yourself and a link to a relevant website. I'll add you as a contributor after this is merged. So excited :)

**Comment by dtsdwarak** [comment]
@wesleykendall Hi Wes! Thanks for reviewing. Guess I've made the changes necessary. Equally excited for this is my first pull request. :smile:

**Pull request merged**